core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/SubmissionCostService.scala

```diff
@@ -80,14 +80,7 @@ class SubmissionCostService(tableName: String, serviceProject: String, billingSe
     val querySql: String =
       generateSubmissionCostsQuery(submissionId, submissionDate, terminalStatusDate)
 
-    val namespaceParam =
-      new QueryParameter()
-        .setParameterType(stringParamType)
-        .setParameterValue(new QueryParameterValue().setValue(workspaceNamespace))
-
-    val queryParameters: List[QueryParameter] = List(namespaceParam)
-
-    executeBigQuery(querySql, queryParameters) map { result =>
+    executeBigQuery(querySql, List.empty) map { result =>
```

Comment:
Out of curiosity, could you leave a comment on what this is doing? I see that it used to be something and now is `Nil`.